src/state/disconnect-toast-mixin.ts

```diff
@@ -35,15 +38,19 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
           action: {
             text:
               this.hass!.localize("ui.notification_toast.dismiss") || "Dismiss",
-            action: () => {},
+            action: () => {
+              this._subscribeBootstrapIntergrations();
```

Comment:
```suggestion
              this._unsubscribeBootstrapIntergrations();
```

Reply:
I just pushed a change to remove it since once we get wrapping up its never going to send any more anyways.